### ISSUE TITLE
Add support for Prime Video

### ISSUE THREAD

**Issue by LivetsHaordaSkola** [spaam/svtplay-dl#1334]
https://www.primevideo.com/detail/0L8K53PWM510Q2Y1NJ1PR9DL4O shows that svtplay-dl does not support to download from that site.
Is it possible to add support for it?

**Comment by qnorsten** [comment]
Both audio and video from primevidee are drm protected, so can't be supported by svtplay-dl

**Comment by andersm1019** [comment]
You can use AnyStream from RedFox.

**Comment by WheelCool** [comment]
> You can use AnyStream from RedFox.

Good idea, but I happen to use Linux and as far as I can see they don't make binaries for Linux.

**Comment by andersm1019** [comment]
> > You can use AnyStream from RedFox.
> 
> Good idea, but I happen to use Linux and as far as I can see they don't make binaries for Linux.

Well, beggars can't be choosers. If you're serious about downloading from Prime Video, I guess you'll have to use one of the available tools. As far as I know, they all run on Windows, unless you can find some secret software via contacts on the "pirate scene".

In any case, don't expect svtplay-dl ever to support Prime Video. That would imply breaking the Widevine DRM-protection as used by most commercial streaming services. Take-down notices and legal threats will be sure to follow...
